web/blueprints/facilities/__init__.py

```diff
@@ -40,6 +42,36 @@ def root():
 def overview():
     return render_template('facilities/site_overview.html')
 
+@bp.route('/overcrowded')
+@nav.navigate(u"Two users one room")
+def overcrowded():
+    return render_template("facilities/room_overcrowded.html")
+
+@bp.route('/overcrowded/json')
+def overcrowded_json():
+    query = session.session.query(User.room_id, func.array_agg(User.id), func.array_agg(User.name))\
```

Comment:
Just querying for the Room ID is not sufficient, we need the full ORM object in order to display a useful string representation. However, something like `Room.q.join(User).group_by(Room.id).having(func.count(User.account_id) > 1).add_columns(func.array_agg(User.id)).all()
` does not work for a curious error („Unhashable type: List“)